src/main/java/com/springboot/soapclient/SoapClientConfig.java

```diff
@@ -0,0 +1,29 @@
+package com.springboot.soapclient;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.oxm.jaxb.Jaxb2Marshaller;
+
+@Configuration
+public class SoapClientConfig{
+
+    @Bean
+    public Jaxb2Marshaller marshaller(){
+        Jaxb2Marshaller marshaller = new Jaxb2Marshaller();
+        // this package must match the package in the specified in
+        // pom.xml
+        marshaller.setContextPath("movies.wsdl");
+        return marshaller;
+    }
+
+    @Bean
+    public MovieClient movieClient(Jaxb2Marshaller marshaller) {
+        MovieClient client = new MovieClient();
+        client.setDefaultUri("http://localhost:8080/ws/movies");
```

Comment:
For this kind of test it is totally fine. However, configs should be put into application.properties and fetched either by property classes or simply by @Value annotation which could define default values too